<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="WKS-CST-19" name="Default test" tests="1" failures="1" timestamp="24 Mar 2016 15:32:37 GMT" time="27.411" errors="0">
  <testcase name="loginTest" time="17.66" classname="ru.greg3d.loginTest">
    <failure type="java.lang.ClassCastException" message="com.sun.proxy.$Proxy12 cannot be cast to org.openqa.selenium.JavascriptExecutor">
      <![CDATA[java.lang.ClassCastException: com.sun.proxy.$Proxy12 cannot be cast to org.openqa.selenium.JavascriptExecutor
	at ru.yandex.qatools.htmlelements.element.SpecCheckBox.cheatClick(SpecCheckBox.java:87)
	at ru.yandex.qatools.htmlelements.element.SpecCheckBox.cheatDeselect(SpecCheckBox.java:83)
	at ru.greg3d.factory.elements.html.FiltersByStatesContainer.deselectActive(FiltersByStatesContainer.java:38)
	at ru.greg3d.applogic.implementations.AllDocumentsPageHelper.testPager(AllDocumentsPageHelper.java:70)
	at ru.greg3d.loginTest.loginTest(loginTest.java:22)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:113)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:206)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:177)
]]>
    </failure>
  </testcase> <!-- loginTest -->
</testsuite> <!-- Default test -->
